flowmachine/tests/test_cache.py

```diff
@@ -70,11 +71,7 @@ def test_store_cache_simple(flowmachine_connect):
     dl1.store().result()
     # Should be stored
     assert dl1.is_stored
-    in_cache = bool(
-        flowmachine_connect.fetch(
-            f"SELECT * FROM cache.cached WHERE query_id='{dl1.md5}'"
-        )
-    )
+    in_cache = cache_table_exists(flowmachine_connect, dl1.md5)
```

Comment:
Thanks for adding the helper function `cache_table_exists()`, it reads a lot cleaner now!

The `in_cache` variable isn't really needed any more though, so maybe change this to `assert cache_table_exists(flowmachine_connect, dl1.md5)`? (Also applies to the other tests.)